src/main/java/com/nylas/Neural.java

```diff
@@ -0,0 +1,105 @@
+package com.nylas;
+
+import okhttp3.HttpUrl;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Neural {
+	protected final String neuralPath = "neural";
```

Comment:
this appears to be a constant, used in only one place.  I suggest either inlining it, or making it a `private static final`